src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -341,6 +342,9 @@ def add_photon_series(
     photon_series_index: int, default: 0
         The metadata for the photon series is a list of the different photon series to add.
         Specify which element of the list with this parameter.
+    parent_container: {'acquisition', 'processing/ophys'}, optional
+        The container where the photon series is added, default is nwbfile.acquisition.
+        When 'processing/ophys' is chosen, the photon series is added to nwbfile.processing['ophys'].
```

Comment:
@h-mayorquin is this clear enough?

Reply:
Sorry for not answering this before, @weiglszonja. For what is worth even if the issue is merged, this looks great to me.

Reply:
thank you! sorry for not pinging before the merge..

Reply:
Not at all, I am glad you guys worked out. I am happy that I could helped you review the other PR and thankful for Cody time on reviewing this one. Seems like a good distribution of work!